values.yaml

```diff
@@ -40,12 +40,20 @@ kpi:
       KPI_DEFAULT_FILE_STORAGE: "storages.backends.s3boto3.S3Boto3Storage"
       # -- Set s3 bucket name is relevant
       KPI_AWS_STORAGE_BUCKET_NAME: "kobo-eks-staging"
+      # -- KoBoCAT Django file storage, should match KPI setting
+      KOBOCAT_DEFAULT_FILE_STORAGE: "storages.backends.s3boto3.S3Boto3Storage"
+      # -- KoBoCAT Bucket name 
+      KOBOCAT_AWS_STORAGE_BUCKET_NAME: "kobo-eks-staging"
```

Comment:
I don't think we want a default here. I would make this # KOBOCAT_AWS_STORAGE_BUCKET_NAME to show that it may need to be set. I see this was already set for KPI but might as well fix it now.